### PR TITLE
Improve our red color palette

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Improve the red color palette](https://github.com/webkom/lego-webapp/commit/cc02f8d0c5a3a2645d5c88a10d1364343ad41ece) 

New palette! No longer pink, less "gray", slightly lighter and works much
better on dark mode. The colors finally conform with our brand guidelines.
Also standardized the danger/error color.

[Here's a Figma ](https://www.figma.com/file/vfSmbQZJpKqoSlodfNWkH3/red?node-id=0%3A1&t=YjEN1kBImfhwek5N-1)of the palette.

---

[Minor improvements to the gallery editor](https://github.com/webkom/lego-webapp/commit/8c2db639e59de931f53cf65894fc7c3e65b3f2bc)

# Result

The changes in the red color palette should be experienced yourself.

---

The changes to the gallery editor is very minimal, and has nothing to do with the red color palette. I just did it because I didn't want to finish my assignment due tomorrow morning.

| Before | After |
:------------------:|:-----------------------------:
image | image
image | image

# Testing

- [x] I have thoroughly tested my changes.

Went through ALL uses of "red".

### CODE REVIEW

**Review by eikhr — APPROVED** [review]
Definitely an improvement! Great job!

I think the new link color is a bit too light in light mode, but that's just my personal preference, no need to change it if you disagree.

**Comment by ivarnakken** [comment]
> I think the new link color is a bit too light in light mode, but that's just my personal preference, no need to change it if you disagree.

The link color is the "default red" color", which is the official Abakus color according to [our brand guidelines](https://abakus.no/brand). I like the new lighter look - it's more modern, and it reminds me of the light Twitter blue and the YouTube red.

File: app/styles/globals.css [hunk 1]
```diff
@@ -27,6 +27,7 @@ a {
   color: var(--lego-link-color);
   cursor: pointer;
   text-decoration: none;
+  transition: color var(--easing-fast);
```
[the remark this answers is not in the document]
Comment (LudvigHz) [reply app/styles/globals.css:30]: I notice now that the slow transition on the checkboxes have been there for a little while as well. They were not as noticeable as all the links I guess, but I still don't dig it :/

I tried with `linear-fast` on the checkboxes and it's an immediate huge improvement for me. Even though they still feel "sluggish".

**Review by LudvigHz — COMMENTED** [review]
Cool!
This is mostly really nice. The only thing that stands out to me immediately is ex. the poll component, where the larger "blocks" are now really red and bright, and I feel like they're a _bit_ too bright. But it might be okay.

The other thing is the suuuper slow link hover transitions. I _strongly_ dislike them with this transition. Some transition might be okay, but right now it's way too slow. I would like no transition at all though :) It's just more snappy and responsive

**Pull request merged**